From 9a7dead72f41e79979625c9bdef2fb638427d3d6 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Mon, 22 Aug 2022 20:54:17 -0700
Subject: [PATCH] wscript: Widen the search for tags

Default is to look for annotated tags, howveer when using devtool we
create our own git tree from release tarballs which will have tags but
they are not annotated, therefore broaden the search to include all tags

Upstream-Status: Inappropriate [OE-specific]

Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 wscript | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/wscript b/wscript
index 879ded1..dff835d 100644
--- a/wscript
+++ b/wscript
@@ -177,7 +177,7 @@ def configure(ctx):
     if build_desc:
         build_desc = ' ' + build_desc
     if ctx.env.BIN_GIT:
-        cmd = ctx.env.BIN_GIT + shlex.split("describe --dirty")
+        cmd = ctx.env.BIN_GIT + shlex.split("describe --tags --dirty")
         git_short_hash = ctx.cmd_and_log(cmd).strip()
         git_short_hash = '-'.join(git_short_hash.split('-')[1:])
 
